public/app/plugins/datasource/graphite/query_ctrl.ts

```diff
@@ -415,6 +429,16 @@ export class GraphiteQueryCtrl extends QueryCtrl {
   getCollapsedText() {
     return this.target.target;
   }
+
+  private handleTagsAutoCompleteError(error: Error): void {
+    console.log(error);
+    if (!this._tagsAutoCompleteErrorShown) {
+      this._tagsAutoCompleteErrorShown = true;
+      dispatch(
+        notifyApp(createErrorNotification(`Fetching tags failed: ${error.message}. Check console for more details.`))
```

Comment:
Is there any details for the user that could be interesting in the console log? I would assume only stack trace which would point to our code where we created the error but that's not going to be cause of the problem anyway. So we can keep the console.log for debugging, pointing it out for the user does not seem like it will give them any more useful information.

Reply:
Yep, the most useful info is in the network tab and I guess users would check it anyway. I'm removing "Check console..." thing 👍